iOS/Flows/People/PeopleCoordinator.swift

```diff
@@ -88,13 +88,13 @@ extension PeopleCoordinator {
     func invite(person: Person) async -> Connection? {
         await self.peopleNavController.peopleVC.showLoading(for: person)
 
-        // This person is not in you contacts, so you can't invite them.
-        guard let contact = person.cnContact else { return nil }
+        // You can't invite a person without a phone number.
+        guard let phoneNumber = person.phoneNumber else { return nil }
 
         // If the user already has an account, we can just connect with them directly.
-        if let user = await self.findUser(with: contact) {
+        if let user = await self.findUser(withPhoneNumber: phoneNumber) {
```

Comment:
In the case of a Jibber User we don't need the contact, we get the phone number directly.